staging/publishing/rules.yaml

```diff
@@ -629,9 +629,23 @@ rules:
       branch: master
       dir: staging/src/k8s.io/kubectl
     name: master
+    dependencies:
+      - repository: apimachinery
+        branch: master
+      - repository: api
+        branch: master
+      - repository: client-go
+        branch: master
   - source:
       branch: release-1.15
       dir: staging/src/k8s.io/kubectl
     name: release-1.15
     go: 1.12.5
 
+    dependencies:
+      - repository: apimachinery
```

Comment:
nit: to be alphabetically consistent, would be better to put `api` over `apimachinery` :stuck_out_tongue:

Reply:
Great catch! Thanks for reviewing, I will change this.